core/app/models/uffizzi_core/credential.rb

```diff
@@ -2,4 +2,16 @@
 
 class UffizziCore::Credential < UffizziCore::ApplicationRecord
   include UffizziCore::Concerns::Models::Credential
+  extend Enumerize
+
+  enumerize :type,
+            in: [
+              UffizziCore::Credential::Amazon.name,
+              UffizziCore::Credential::Azure.name,
+              UffizziCore::Credential::DockerHub.name,
+              UffizziCore::Credential::DockerRegistry.name,
+              UffizziCore::Credential::Github.name,
```

Comment:
I think we could not use Github credentials in core

Reply:
The thing is that there are test that create github credentials, if I don't add this those test fails!

This one for example `core/test/controllers/uffizzi_core/api/cli/v1/projects/deployments_controller_test.rb`

```
@credential = create(:credential, :github, :active, account: @account, provider_ref: generate(:number))
```

Reply:
Github credentials removed and test updated